### PR TITLE
fix(config): make promise tests actually test the full promise chain

### PR DESCRIPTION
**Pull request by diasdavid**
all we need for `js-ipfs 0.21`:)

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -158,12 +158,11 @@ module.exports = (common) => {
...
-            .then(() => {
-              ipfs.config.get('Fruit', (err, fruit) => {
-                expect(err).to.not.exist
-                expect(fruit).to.equal('banana')
-              })
+            .then(() => ipfs.config.get('Fruit'))
+            .then((fruit) => {
+              expect(fruit).to.equal('banana')
             })
+            .catch((err) => expect(err).to.not.exist)
```
Comment (VictorBjelkholm) [inline src/config.js]: Shouldn't need the catch here since we're returning the promise so mocha should automatically catch any errors.

Comment (diasdavid) [inline src/config.js]: right

Comment (diasdavid) [inline src/config.js]: fixed :)

**Comment by diasdavid** [comment]
Thank you for the CR @VictorBjelkholm :)

**Pull request merged**